pandas/tests/window/test_rolling.py

```diff
@@ -121,35 +121,58 @@ def test_numpy_compat(method):
         getattr(r, method)(dtype=np.float64)
 
 
-def test_closed_fixed(closed, arithmetic_win_operators):
+# maybe this one too?
+@pytest.mark.parametrize(
+    "center",
```

Comment:
I think we have a pytest fixture for this already.

Reply:
Good hint, thanks! We used the fixture now instead.